terraform/modules/front_door/cloudfront.tf

```diff
@@ -0,0 +1,58 @@
+locals {
+  origin_id = "${var.prefix}-origin"
+}
+
+#tfsec:ignore:aws-cloudfront-enable-logging:TODO we will be implementing logging later
+#tfsec:ignore:aws-cloudfront-enable-waf:TODO CLDC-2546
+#tfsec:ignore:aws-cloudfront-enforce-https:TODO CLDC-2654
+#tfsec:ignore:aws-cloudfront-use-secure-tls-policy:TODO CLDC-2680
+resource "aws_cloudfront_distribution" "this" {
+  #checkov:skip=CKV_AWS_34:TODO CLDC-2654
+  #checkov:skip=CKV2_AWS_42:TODO CLDC-2680
+  #checkov:skip=CKV2_AWS_47:TODO CLDC-2546 when setting up WAF it should be configured appropriately to mitigate against the Log4j vulnerability https://docs.bridgecrew.io/docs/ensure-aws-cloudfront-attached-wafv2-webacl-is-configured-with-amr-for-log4j-vulnerability
+  #checkov:skip=CKV_AWS_68:TODO CLDC-2546
+  #checkov:skip=CKV_AWS_86:TODO we will be implementing logging later
+  #checkov:skip=CKV_AWS_174:TODO CLDC-2680
+  #checkov:skip=CKV_AWS_305:no need to define a default root object because the root of our distribution is just the app's homepage
+  enabled         = true
+  is_ipv6_enabled = true
+  price_class     = "PriceClass_100" # Affects which edge locations are used by cloudfront, which affects the latency users will experience in different geographic areas
+
+  origin {
+    domain_name = aws_lb.main.dns_name
+    origin_id   = local.origin_id
+
+    custom_origin_config {
+      http_port                = 80
+      https_port               = 443
+      origin_keepalive_timeout = 60
+      origin_protocol_policy   = "match-viewer"
```

Comment:
Should this be "http-only" for now?

Reply:
I guess so - our load balancer can't accept https yet.